python/test/unit/language/test_core.py

```diff
@@ -3366,9 +3366,16 @@ def get_test_dot_small_mn_fma_cases():
             for in_dtype, out_dtype in [('float16', 'float16'), ('float32', 'float32')]]
 
 
+def get_test_dot_double_rate_cases():
```

Comment:
Can we move this into the AMD test folder?

Reply:
This one is fine here given we share the test between CUDA and HIP.